app/views/users/_address.html.erb

```diff
@@ -1,4 +1,13 @@
 <h1><%= @user.name %></h1>
 <p><%= @user.email %></p>
-<p><%= @user.address %></p>
-<p><%= @user.city %> <%= @user.state %> <%= @user.zip %></p>
+  <% @user.addresses.each do |address| %>
+  <section id="address-<%= address.id %>">
+    <p><%= address.address %></p>
+    <p><%= address.city %> <%= address.state %> <%= address.zip %></p>
+    <%= link_to 'Edit This Address', edit_user_address_path(address) %>
+      <% if address.shipped_orders.any? %>
```

Comment:
What you really care about here is whether or not there are any shipped orders, so I would make the shipped_orders method return a boolean, rather than an array of addresses that you then have to call .any? on. This is a case where you want to build the method based on how you are going to use it.